### PR TITLE
Rc14

### PR DESCRIPTION
**Pull request by pirog**
Thank you so much for contributing code to Lando! If this is your **first time** contributing to Lando we recommend you check out

* [Contributing to Lando](https://docs.devwithlando.io/contrib/contributing.html)
* [Lando Developer Guide](https://docs.devwithlando.io/dev/started.html)

We will get to your PR as soon as we can but in the meantime you might want to check to make sure you have done the following. **The more boxes you can check the easier it will be for us to merge in your changes with confidence**

- [ ] My code includes the latest code from the `master` branch
- [ ] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/docs/changelog)
- [ ] My code includes a [functional test](https://docs.devwithlando.io/dev/testing.html#functional-tests) if applicable
- [ ] My code includes a [unit test](https://docs.devwithlando.io/dev/testing.html#unit-tests) if applicable
- [ ] My code includes documentation updates if applicable.
- [ ] My code passes relevant CI status checks
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

File: docs/tutorials/joomla.md [hunk 1]
```diff
@@ -147,25 +149,33 @@ You may need to override our [default Joomla config](https://github.com/lando/la
...
+   |-- server.conf
 |-- index.php
 |-- .lando.yml
 ```
 
 **Landofile using custom joomla config**
 
 ```yaml
-recipe: joomla
+recipe: drupal7
```
Comment (uberhacker) [inline docs/tutorials/joomla.md]: Why was this changed?

Comment (pirog) [reply docs/tutorials/joomla.md]: copy+paste failure, thanks @uberhacker

File: plugins/lando-core/index.js [hunk 2]
```diff
@@ -15,8 +15,6 @@ const defaults = {
       LANDO: 'ON',
       LANDO_WEBROOT_USER: 'www-data',
       LANDO_WEBROOT_GROUP: 'www-data',
-      LANDO_WEBROOT_UID: '33',
-      LANDO_WEBROOT_GID: '33',
```
Comment (uberhacker) [inline plugins/lando-core/index.js]: :+1:

File: plugins/lando-core/scripts/user-perms.sh [hunk 3]
```diff
@@ -22,8 +22,8 @@ fi
 # Set defaults
 : ${LANDO_WEBROOT_USER:='www-data'}
 : ${LANDO_WEBROOT_GROUP:='www-data'}
-: ${LANDO_WEBROOT_UID:='33'}
-: ${LANDO_WEBROOT_GID:='33'}
+: ${LANDO_WEBROOT_UID:=$(id -u $LANDO_WEBROOT_GROUP)}
+: ${LANDO_WEBROOT_GID:=$(id -g $LANDO_WEBROOT_GROUP)}
```
Comment (uberhacker) [inline plugins/lando-core/scripts/user-perms.sh]: :+1:
